framework/filters/AjaxFilter.php

```diff
@@ -48,7 +48,7 @@ class AjaxFilter extends ActionFilter
     public function init()
     {
         if ($this->request === null) {
-            $this->request = Yii::$app->getRequest();
+            $this->request = Yii::get('request');
```

Comment:
What's the use case for a configurable `Request` object?
 (Understand if you have to keep it for backwards compatibility)